src/DateTimeParser.jl

```diff
@@ -152,25 +167,26 @@ function _parsedate(datetimestring::AbstractString; fuzzy::Bool=false,
             # Token is a number
             i += 1 # We want to look at what comes after the number
             if length(ymd) == 3 && tokenlength in (2,4) &&
-                (i>=len || (tokens[i] != ":" && !haskey(HMS, lowercase(tokens[i]))))
+                (i>=len || (tokens[i] != ":" && !haskey(HMS[locale], lowercase(tokens[i]))))
```

Comment:
Condition can be changed to `length(ymd) == 3 && tokenlength in (2,4) &&i >= len` without causing the testcases to fail. Add additional tests?